Gruntfile.js

```diff
@@ -481,6 +481,13 @@ module.exports = (grunt) => {
     done()
   })
 
+  grunt.registerTask('pin', function (version) {
+    if (typeof version !== 'string') throw new Error('usage: grunt pin:<version>')
+
+    grunt.task.run('build')
+    grunt.task.run(`_pin:${version}`)
```

Comment:
created an internal task `_pin` to make sure `build` runs first before executing `pin`
(adding `grunt.task.run('build')` inside the original `pin` custom task actually queues this task at the end of it. but pls let me know if there is a better way)